src/main/kotlin/no/nav/syfo/personstatus/db/createOrUpdatePersonOversiktStatus.kt

```diff
@@ -155,9 +155,9 @@ fun Connection.updatePersonOversiktStatusOppfolgingstilfelle(
 }
 
 fun DatabaseInterface.lagreBrukerKnytningPaEnhet(veilederBrukerKnytning: VeilederBrukerKnytning) {
-    val id = oppdaterEnhetDersomKnytningFinnes(veilederBrukerKnytning)
+    val exists = oppdaterVeilederDersomBrukerFinnes(veilederBrukerKnytning)
```

Comment:
Synes refaktoreringen av metoden er fin, men vet ikke om det gir mening ut i fra navnet at `oppdaterVeilederDersomBrukerFinnes` returnerer en boolsk verdi, og at verdien burde hete `exists`. 

Hva som den bare returnerer `rowsUpdated: Int`, og så sjekker vi `rowsUpdated` i if-en?

```kotlin
val rowsUpdated = oppdaterVeilederDersomBrukerFinnes(veilederBrukerKnytning)

if (rowsUpdated == 0) { 
  ...
}
```

Reply:
Ja, kan sjekke om det blir bedre (evnt slå sammen de to metodene siden det ikke er så mye kode igjen i lagreBrukerKnytningPaEnhet nå).